alloc-fmt/README.md

```diff
@@ -0,0 +1,16 @@
+<!-- Copyright 2017 the authors. See the 'Copyright and license' section of the
+README.md file at the top-level directory of this repository.
+
+Licensed under the Apache License, Version 2.0 (the LICENSE-APACHE file) or
+the MIT license (the LICENSE-MIT file) at your option. This file may not be
+copied, modified, or distributed except according to those terms. -->
+
+alloc-fmt
+=========
+
+`alloc-fmt` provides formatting and assertion macros similar to `println`,
```

Comment:
Maybe reword this to indicate that when you say "in an allocator" you mean "global allocator" in particular. I think it's clear from context but we shouldn't give the impression that more specific allocators should worry about println